tdrs-backend/docker-compose.yml

```diff
@@ -28,10 +30,14 @@ services:
     environment:
       - SERVICES=s3
       - DATA_DIR=/tmp/localstack/data
+      - AWS_BUCKET
+      - AWS_REGION_NAME
     ports:
       - "4566:4566"
     volumes:
       - localstack_data:/tmp/localstack
+      # Copy in the Localstack setup script to configure any buckets needed
+      - ./localstack-setup.sh:/docker-entrypoint-initaws.d/localstack-setup.sh
```

Comment:
Per the [localstack docs](https://github.com/localstack/localstack#initializing-a-fresh-instance) this is the recommended way to initialize localstack. Any scripts placed in `docker-entrypoint-initaws.d` will be run after all localstack services have started.